CSharp/Library/Microsoft.Bot.Connector.Shared/ConnectorAPI/Models/ActionTypes.cs

```diff
@@ -0,0 +1,27 @@
+// <auto-generated>
```

Comment:
Won't the lines in generateClient.cmd (in BotBuilder) that say "call ..\node_modules\.bin\replace "using Models;" "" . -r --include="*.cs""  cause build errors if we move these to Models?   Should we update that script as well? Or leave these back in the ConnectorAPI directory?